### PR TITLE
Add Jordan holidays

### PR DESCRIPTION
**Pull request by yousefdev20**

### CODE REVIEW

File: src/Countries/Jordan.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+use Carbon\CarbonImmutable;
+
+class Jordan extends Country
+{
+    public function countryCode(): string
+    {
+        return 'jo';
+    }
+
+    /** @return array<string, CarbonImmutable> */
```
Comment (Nielsvanpach) [inline src/Countries/Jordan.php:14]: can be removed

File: src/Countries/Jordan.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        return array_merge([
+            'New Year\'s Day' => '01-01',
+            'Labor' => '05-01',
+            'Independence Day' => '05-25',
+            'Christmas Day' => '12-25',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
```
Comment (Nielsvanpach) [inline src/Countries/Jordan.php:26]: If there are no variable holidays in Jordan, no need for this function.

File: tests/.pest/snapshots/Countries/JordanTest/it_can_calculate_jordanian_holidays.snap [hunk 3]
```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "name": "New Year's Day",
```
Comment (Nielsvanpach) [inline tests/.pest/snapshots/Countries/JordanTest/it_can_calculate_jordanian_holidays.snap:3]: I assume there are more holidays in Jordan?

Comment (ahmadabuaysheh) [reply tests/.pest/snapshots/Countries/JordanTest/it_can_calculate_jordanian_holidays.snap:3]: Other holidays are calculated based on the lunar calendar.
Which we convert to gregorian calendar when needed.
Actually, many countries have holidays based on this type of calendar; therefore, we might need to update the core of the app to support it.

Comment (Nielsvanpach) [reply tests/.pest/snapshots/Countries/JordanTest/it_can_calculate_jordanian_holidays.snap:3]: You can look at Turkey: https://github.com/spatie/holidays/blob/main/src/Countries/Turkey.php#L11

**Comment by Nielsvanpach** [comment]
Thanks, but I'm closing this PR as it doesn't feel complete yet. Feel free to reopen once fixed!

**Pull request closed without merging**